site/content/help/quickstart/index.md

```diff
@@ -30,36 +30,36 @@ description: A guide designed to get you started with yuzu quickly.
 
 #### CPU: 
 
-Any x86_64 CPU with support for the FMA instruction set. 6 threads or more are recommended.
+Any x86_64 CPU with support for the FMA and AVX2 instruction set. 6 threads or more are recommended. Pre-Ryzen CPUs are strongly not recommended.
 
 - Minimum: Intel Core i5-4430 / AMD Ryzen 3 1200
 
 - Recommended: Intel Core i5-10400 / AMD Ryzen 5 3600
 
 #### Dedicated graphics: 
 
-OpenGL 4.6 or Vulkan 1.1 compatible hardware and drivers are mandatory. Half-float support and 4GB of VRAM are recommended.
+OpenGL 4.6 or Vulkan 1.2 compatible hardware and drivers are mandatory. Half-float and VP9 decoding support, are recommended. At native resolutions, 4GB of video memory (VRAM) is strongly recommended. Vulkan 1.3 support is strongly recommended.
 
 - Minimum for Linux: NVIDIA GeForce GT 1030 2GB / AMD Radeon R7 240 2GB
 
 - Minimum for Windows: NVIDIA GeForce GT 1030 2GB / AMD Radeon RX 550 2GB
 
-- Recommended: NVIDIA GeForce GTX 1650 4GB / AMD Radeon RX Vega 56 8GB
+- Recommended: NVIDIA GeForce GTX 1660 6GB / AMD Radeon RX 6600 8GB
 
 #### Integrated graphics:
 
 Integrated graphics will produce very low performance. A dedicated GPU will produce better results on all scenarios.
 This is only for listing iGPU support.
 
-- Minimum for Linux: Intel HD 5300 / AMD Radeon R5 Graphics
+- Minimum for Linux: Intel HD 4400 / AMD Radeon Vega 3
 
-- Minimum for Windows: Intel HD Graphics 520 / AMD Radeon Vega 3
+- Minimum for Windows: Intel UHD Graphics 730 / AMD Radeon Vega 3
 
-- Recommended: Intel UHD Graphics 750 / AMD Radeon Vega 7
+- Recommended: Intel Xe Graphics / AMD Radeon Graphics (Vega 8)
 
 #### RAM: 
 
-Since an integrated GPU uses system RAM as its video memory (VRAM), our memory requirement in this configuration is higher.
```

Comment:
is this change really needed?

Reply:
Reordered where what VRAM means so it got modified.